Packs/Base/Scripts/CommonServerPython/CommonServerPython.yml

```diff
@@ -16,6 +16,7 @@ timeout: 0s
 alt_dockerimages: # used for unit testing
 - demisto/python3:3.7.3.286
 - demisto/python3:3.8.3.8494
+- demisto/python3:3.9.1.14969
```

Comment:
@anara123 @yaakovi @Itay4 note that I also added to run unit testing on py 3.9.1